_includes/pbc/all/install-features/202307.0/install-the-shipment-customer-account-management-feature.md

```diff
@@ -0,0 +1,246 @@
+
+Add `ShipmentTypeAddressFormWidget` to the `address` view of `CheckoutPage` module.
+
+```twig
+{% raw %}{% widget 'ShipmentTypeAddressFormWidget' args [data.form] with {
+    data: {
+        deliveryContainerClassName: deliveryContainerClassName,
+        billingSameAsShippingContainerClassName: embed.jsAddressClass ~ '__wrapper-billingSameAsShipping',
+    },
+} only %}
+{% endwidget %}{% endraw %}
+```
+
+{% info_block infoBox "Info" %}
+
+`deliveryContainerClassName` - class name of container the delivery form and the address selector.<br>
+`embed.jsAddressClass ~ '__wrapper-billingSameAsShipping'` - class name of container for billing same as shipping checkbox.
+
+{% endinfo_block %}
+
+2. Multi shipment.
+
+{% info_block infoBox "Info" %}
+
+You can skip this step if multi-shipment isn't used on the project.
+
+{% endinfo_block %}
+
+a) Add the same widget as for single shipment.
+
+b) Add `multiple-shipment-toggler` molecule to the `address` view of `CheckoutPage` module.
+
+```twig
+{% raw %}{% include molecule('multiple-shipment-toggler', 'CheckoutPage') with {
+    data: {
+        isMultipleShipmentSelected: isMultipleShipmentSelected,
+    },
+    attributes: {
+        'toggle-targets-class-name': singleDeliveryContainerClassName,
+        'select-class-name': deliverySelectClassName,
+    },
+} only %}{% endraw %}
+```
+
+{% info_block infoBox "Info" %}
+
+`isMultipleShipmentSelected` - flag that indicates if multiple shipment is selected.<br>
+`singleDeliveryContainerClassName` - class name of container the address selector and the `ShipmentTypeAddressFormWidget`.<br>
+`deliverySelectClassName` - class name of the address selector.
+
+{% endinfo_block %}
+
+c) Add `ShipmentTypeAddressFormWidget` to the `address-item-form-field-list` molecule of `CheckoutPage` module.
+
+```twig
+{% raw %}{% widget 'ShipmentTypeAddressFormWidget' args [item] with {
+    data: {
+        deliveryContainerClassName: deliveryContainerClassName,
+        shipmentTypesClassName: data.validatorTriggerClassName,
+        servicePointClassName: data.itemShippingClassName,
+    },
+} only %}{% endwidget %}{% endraw %}
+```
+
+d) Adjust `address-item-form` molecule in `CheckoutPage` module by adding `extra-triggers-class-name` attributes propery for `validate-next-checkout-step` molecule to validate `pickup` shipment type.
+
+```twig
+{% raw %}{% include molecule('validate-next-checkout-step', 'CheckoutPage') with {
+    class: validatorClassName,
+    attributes: {
+        'container-selector': '.' ~ itemShippingClassName,
+        'target-selector': '.' ~ data.jsAddressClass ~ '__form-submit',
+        'dropdown-trigger-selector': '.' ~ addressSelectClassName ~ ':not(.' ~ data.hiddenClassName ~ ')',
+        'extra-triggers-class-name': validatorTriggerClassName,
+        'parent-target-class-name': config.jsName ~ '__items',
+        'is-enable': false,
+    },
+} only %}{% endraw %}
+```
+
+3. Build assets.
+
+Enable Javascript changes:
```

Comment:
same: If building assets step consists of the only step "Enable Javascript changes", accept this suggestion:
```suggestion
```